modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/ExplainRelAsTextWriter.java

```diff
@@ -78,6 +79,11 @@ protected void explain_(RelNode rel, List<Pair<String, @Nullable Object>> values
             if (value.right instanceof RelNode) {
                 continue;
             }
+            if (rel instanceof IgniteMergeJoin) {
```

Comment:
I think, It's better to adjust `org.apache.ignite.internal.sql.engine.rel.IgniteMergeJoin#explainTerms` in the following way:
```
    @Override
    public RelWriter explainTerms(RelWriter pw) {
        return super.explainTerms(pw)
                .itemIf("leftCollation", leftCollation, !ExplainUtils.forExplain(pw))
                .itemIf("rightCollation", rightCollation, !ExplainUtils.forExplain(pw));
    }
```